packages/codegen/generate.hooks.ts

```diff
@@ -224,6 +227,17 @@ export function generateReactHooks<
         const typedFetchMore: IFetchMoreType<GQLObject, QueryVariables> = fetchMore
         const typedStopPolling: IStopPollingType = stopPolling
 
+        const fetchAll = options && options.fetchAll
```

Comment:
Probably, you need another new method `useAllObjects`. This hack is not safe for rerenders. It's better to avoid redundant rerenders and state changes

Reply:
done